ledger-core-ethereum/src/ethereum/ERC20/ERC20LikeOperationQuery.cpp

```diff
@@ -53,5 +52,18 @@ namespace ledger {
                 .outerJoin("erc20_operations AS e", "o.uid = e.ethereum_operation_uid")
                 .execute(sql);
         }
+
+        void ERC20LikeOperationQuery::inflateCompleteTransaction(
+            soci::session &sql,
+            const std::string &accountUid,
+            ERC20LikeOperation& operation
+        ) {
```

Comment:
So, why is this left empty, you might ask. It’s left empty because `performExecute` is overriden to return the right data. It’s a special case of tokens and we ought to change the whole design of the `OperationQuery` interface later.